multi.lic

```diff
@@ -37,7 +34,20 @@ class Multi
         }
       }
     end
+  end
 
+  def clean_arguments(args)
+    # If this script is invoked from DRC.wait_for_script_to_complete
+    # then the arguments may be enclosed in literal double quotes.
+    # That breaks the parsing logic so if we detect that situation
+    # we strip the leading and trailing double quote.
+    if args =~ /^\"(.*)\"$/
```

Comment:
Since you've defined a regex capture group, couldn't you use that instead of `slice`?

Reply:
Doh, yes, thank you!